view.js

```diff
@@ -1,19 +1,35 @@
+$(document).ready(function (){
+    $.get("viewdatabase.php", function (data){
+        var myJSON = JSON.parse(data);
```

Comment:
Make this variable global to be able to access it from other functions for sorting purpose